data.sql

```diff
@@ -4,3 +4,10 @@ insert into animals(id, name, date_of_birth, escape_attempts, neutered, weight_k
 insert into animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) values (2, 'Gabumon', '2018-11-15', 2, true, 8);
 insert into animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) values (3, 'Pikachu', '2021-01-07', 1, false, 15.04);
 insert into animals(id, name, date_of_birth, escape_attempts, neutered, weight_kg) values (4, 'Devimon', '2017-05-12', 5, true, 11);
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_Kg) values ('Charmander', '2020-02-08', 0, false, 11);
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_Kg) values ('Plantmon', '2021-11-15', 2, true, 5.7);
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_Kg) values ('Squirtle', '1993-04-02', 3, false, 12.13);
+insert into animals (name, date_of_birth, escape_attempts, neutered, weight_Kg) values ('Angemon', '2005-06-12', 1, true, 45);
```

Comment:
- [ ] Great job so far, 👏🏻  However, you need to pay more attention to the project requirements, actually the fourth first insert should have `weight_kg` with a negative value . kindly correct this 👍🏻  Your code should look like this:

```sql
    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Charmander', '2020/2/8', '0', 'false', '-11.0');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Plantmon', '2021/11/15', '2', 'true', '-5.7');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Squirtle', '1993/04/02', '3', 'false', '-12.13');

    INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
    VALUES ('Angemon', '2005/06/12', '1', 'true', '-45.0');
```